deployment.Rmd

```diff
@@ -58,6 +58,22 @@ library(sparklyr)
 sc <- spark_connect(master = "yarn-client")
 ```
 
+or for more customized configuration:
+
+```{r}
+library(sparklyr)
+sc <- spark_connect(master="yarn",
+                    config = list(
```

Comment:
You set the R_CONFIG_ACTIVE environment variable to the name of the active
config:

https://github.com/rstudio/config/blob/master/R/get.R#L26

If it's not set then it uses "default". The idea is that e.g. your
production server would define this to someting else in Rprofile.site or
Renviron.site (i.e. it's ambient value that isn't set or determined within
code, but rather via the configuration of the server it's running on.


On Tue, Aug 9, 2016 at 1:41 PM, Marcin Kosiński <notifications@github.com>
wrote:

> In deployment.Rmd
> <https://github.com/rstudio/spark.rstudio.com/pull/7#discussion_r74107082>
> :
>
> > @@ -58,6 +58,22 @@ library(sparklyr)
> >  sc <- spark_connect(master = "yarn-client")
> >  ```
> >
> > +or for more customized configuration:
> > +
> > +```{r}
> > +library(sparklyr)
> > +sc <- spark_connect(master="yarn",
> > +                    config = list(
>
> Let's say we'll have one config.yml with 3 profiles. How can then I refine
> the specific profile?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/rstudio/spark.rstudio.com/pull/7/files/2bf081b64d28d2c2535c0450770bccf0dc52e926#r74107082>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGXxy-eAE0I0UfReiB_3DDYhmZ3923fks5qeLuzgaJpZM4JgRvE>
> .
>